docs/development/new-cloud-provider.md

```diff
@@ -0,0 +1,65 @@
+# Adding Cloud Providers
+
+This document provides an overview of how to integrate a new cloud provider into Gardener. Each component that requires integration has a detailed description of how to integrate it and the steps required.
+
+## Cloud Components
+
+Gardener is composed of 2 or more Kubernetes clusters:
+
+* Shoot: These are one more "workload" clusters, the regular Kubernetes clusters you have seen. They provide places for your workloads to run.
```

Comment:
s/These are one more "workload" clusters/These are the end-user clusters

Reply:
OK